src/graphql/mutations.js

```diff
@@ -159,26 +159,17 @@ export const createApplicationMutation = gql`
   mutation createApplication($application: ApplicationInput!) {
     createApplication(application: $application) {
       id
+      type
       name
       description
       callbackUrl
+      apiKey
       clientId
       clientSecret
```

Comment:
Should we remove `clientId` and `clientSecret` for now?

Reply:
Indeed, I checked the code and it seems that we only use `id`, `type` and `apiKey`. PR updated.